addons/im_livechat/static/src/embed/cors/persona_model_patch.js

```diff
@@ -0,0 +1,24 @@
+/* @odoo-module */
+
+import { Persona } from "@mail/core/common/persona_model";
+
+import { patch } from "@web/core/utils/patch";
+import { url } from "@web/core/utils/urls";
+
+patch(Persona.prototype, {
+    get avatarUrl() {
+        const param = {
+            guest_token: this._store.env.services["im_livechat.livechat"].guestToken,
+        };
+        if (!param.guest_token && this.type === "partner") {
+            return url(`/im_livechat/operator/${this.id}/avatar`);
```

Comment:
no cors here?

Reply:
The original URL didn't have CORS, I guess it can't hurt to add it now